### ISSUE TITLE
duplication

### ISSUE THREAD

**Issue by bmcage** [wubbahed/lithophane#1]
Ok, not an issue, but I don't find how to send a message to you otherwise.
Perhaps interesting to join forces with https://github.com/MarkDurbin104/3dp.rocks which also generates lithopanes with javascript from ?

**Comment by wubbahed** [comment]
Good catch!  I've reached out to @MarkDurbin104 so let's see what we can make.
